src/main/kotlin/no/nav/syfo/dialogmotekandidat/api/DialogmotekandidatApi.kt

```diff
@@ -41,7 +42,8 @@ fun Route.registerDialogmotekandidatApi(
                 val oppfolgingstilfelleStart = oppfolgingstilfelle?.tilfelleStart
 
                 val kandidatDTO = if (kandidatDate != null && oppfolgingstilfelleStart != null &&
-                    kandidatDate.isAfterOrEqual(oppfolgingstilfelleStart)
+                    kandidatDate.isAfterOrEqual(oppfolgingstilfelleStart) &&
+                    kandidatDate.isBeforeOrEqual(LocalDate.now().minusDays(7))
                 ) {
```

Comment:
Synes denne var litt vanskelig å lese. Kanskje det hadde blitt lettere med en hjelpefunksjon? 🤔 Er det vi lurer på her om det er 7 eller mer, dager siden starten på oppfølgingstilfellet? 😊

Reply:
Vi lurer på om personen ble kandidat etter starten på oppfølgingstilfellet og om det har gått sju dager siden personen ble kandidat. Kan forsøke å forbedre!

Reply:
Trakk det ut i noen variabler for å gjøre det mer forståelig :+1:

Reply:
Nydelig! 😍🤌